agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/service/AGMDiseaseAnnotationService.java

```diff
@@ -63,6 +63,9 @@ public List<AGMDiseaseAnnotation> getFiltered() {
 				}
 			}
 
+			if (page == 0) {
+				display.startProcess("Pulling AGM DA's from curation");
+			}
 			display.progressProcess(response.getReturnedRecords().longValue());
 
 			pages = (int) (response.getTotalResults() / batchSize);
```

Comment:
I think you're still going to run into problems here as response.getTotalResults() is null due to batchSize (limit) not being 0.

Reply:
Ok this I changed how the logic works now.